src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -130,6 +130,12 @@ async function initializeConnection(options: CircuitStartOptions, logger: Logger
     await connection.start();
   } catch (ex) {
     unhandledError(connection, ex, logger);
+
+    if (ex.message.includes('UnsupportedTransportWebSocketsError')) {
+      showErrorNotification('Unable to connect, please ensure you are using an updated browser that supports WebSockets.');
```

Comment:
> Isn't it more likely that the server does not support websockets rather than the browser or a proxy blocking it?

Yes, however in that case there isn't really much the end user can do. Would the standard `Unhandled exception occurred` error banner be sufficient for that case?

For developers the console will have:

```
Error: Error: Unable to connect to the server with any of the available transports. LongPolling failed: Error: 'LongPolling' is disabled by the client.
```

Reply:
More generally, we don't currently show the `Unhandled Exception Occurred` banner when [`connection.start()`](https://github.com/dotnet/aspnetcore/pull/34644/files#diff-7c47c684c78e3f5b5eeecb28b249b03afcefcd90a925cc20b69ec2cdec261752R131) throws. Do we want to maintain this behavior or always show the banner when `connection.start()` throws here?

Reply:
> Would the standard Unhandled exception occurred error banner be sufficient for that case?

I guess. The error is primarily developer focused. 

> More generally, we don't currently show the Unhandled Exception Occurred banner when connection.start() throws.

That seems like a good change. It's another failure mode that a user is made aware of.

Reply:
1c3e8f69ba

Reply:
> > visit https://aka.ms/blazor-server-websockets-error.');
> 
> Where the short link takes you to a section in our deployment docs that talks about the breaking change and how to configure Azure App Services.

I'll link to https://docs.microsoft.com/en-us/aspnet/core/signalr/configuration?tabs=dotnet#advanced-http-configuration-options for now and update the link once we have the docs on the breaking change in place.